src/index.html

```diff
@@ -10,7 +10,58 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      action="/"
+      method="get"
+      data-qa="big"
+    >
+      <div class="search-box">
+        <div class="search-icon">
+          <img
+            class="search-logo"
```

Comment:
Do you really need so many div wrappers here? 
I think it's better to have structure like this:
```
<label>
  <img>
  <input>
</label>
```